src/pybind/mgr/dashboard/tools.py

```diff
@@ -883,7 +883,7 @@ def cors_tool():
             resp_head['Access-Control-Allow-Origin'] = req_header_origin_url
         ac_method = req_head.get('Access-Control-Request-Method', None)
 
-        allowed_methods = ['GET', 'POST', 'PUT']
+        allowed_methods = ['GET', 'POST', 'PUT', 'DELETE']
```

Comment:
since there is another PR, we can remove this and let's merge the other oine

Reply:
Removed